lifetimes/utils.py

```diff
@@ -68,6 +69,11 @@ def calibration_and_holdout_data(
     monetary_value_col: string, optional
         the column in transactions that denotes the monetary value of the transaction.
         Optional, only needed for customer lifetime value estimation models.
+    include_first_transaction: bool, optional
+        Default: False
+        By default the first transaction is not included while calculating frequency and
+        monetary_value. Can be set to True to include it.
+        Should be False if you are going to use this data with any fitters in lifetimes package
```

Comment:
Again, because these functions are quite useful, not just for the purposes of this library. I started using your library for CLV predictions but now I'm using other techniques such as RFM analysis & scoring/segmentation where I've to do the same thing but just include the first transaction too.

Reply:
hm, interesting. I worry that it'll confuse users solely using lifetimes. OTOH, if ppl find it useful, okay then